.github/workflows/pr.yaml

```
@@ -0,0 +1,42 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+name: pr
+
+on:
+  push:
+    branches:
+      - "pull-request/[0-9]+"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-builder:
+    needs:
+      - checks
+      - wheel-build
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/pr-builder.yaml@branch-23.12
+  checks:
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@branch-23.12
+    with:
+      enable_check_generated_files: false
+  wheel-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.10, 3.11, 3.12]
```

FYI, you have to wrap 3.10 in quotes because it's currently being interpreted as a float 3.1.
```suggestion
        python-version: ["3.10", "3.11", "3.12"]
```